app_backend/.env.sample

```diff
@@ -1,2 +1,5 @@
 DB_USERNAME="YOUR_USERNAME"
 DB_PASSWORD="YOUR_PASSWORD"
+
+SECRET="A SECRET KEY"
```

Comment:
Is this an Easter egg?

Reply:
This is for the token stuff. It needs a secret key to encode it